services/venice-server/src/main/java/com/linkedin/venice/listener/ServerReadMetadataRepository.java

```diff
@@ -66,6 +66,11 @@ public MetadataResponse getMetadata(String storeName) {
     MetadataResponse response = new MetadataResponse();
     try {
       Store store = storeRepository.getStoreOrThrow(storeName);
+      if (!store.isStorageNodeReadQuotaEnabled()) {
+        throw new UnsupportedOperationException(
+            String
+                .format("Fast client is not enabled for store: %s, please contact Venice team for support", storeName));
```

Comment:
Can you remove or rephrase the second part "please contact Venice team for support"? As a open-source project, I think we should avoid using `venice team` in the codes/exception msgs. How about  "Fast client is not enabled for store: %s, please contact the service team to enable it"?

Reply:
Good call, let me change that